package.json

```diff
@@ -78,8 +78,8 @@
   "author": "kiwi.com",
   "license": "MIT",
   "peerDependencies": {
-    "react": "^16.8.3",
-    "react-dom": "^16.8.3",
+    "react": "^16.9.0",
```

Comment:
Please don't change or add dependencies when it's not necessary. We don't even know if it can be raised due to main repo, where `16.8.3` is used also - update needs to be done there first.

Reply:
that's not a major version, it will not break something for sure.